<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>850</width>
    <height>850</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1500</width>
    <height>1500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>320</y>
      <width>271</width>
      <height>132</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <item row="1" column="1" alignment="Qt::AlignHCenter">
      <widget class="QSpinBox" name="spinBoxHeigth">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);
</string>
       </property>
       <property name="minimum">
        <number>200</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0" alignment="Qt::AlignLeft">
      <widget class="QLabel" name="labelHeigth">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
font: 75 14pt &quot;Height&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Gamearea heigth:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" alignment="Qt::AlignHCenter">
      <widget class="QSpinBox" name="spinBoxWidth">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="minimum">
        <number>200</number>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="singleStep">
        <number>10</number>
       </property>
       <property name="value">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelWidth">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
font: 75 14pt &quot;Height&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Gamearea width:</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" alignment="Qt::AlignHCenter">
      <widget class="QSpinBox" name="spinBoxSeed">
       <property name="maximumSize">
        <size>
         <width>16777212</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
       <property name="value">
        <number>1</number>
       </property>
      </widget>
     </item>
     <item row="3" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelSeed">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
font: 75 14pt &quot;Height&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Seed value:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0" alignment="Qt::AlignRight">
      <widget class="QLabel" name="labelUsername">
       <property name="styleSheet">
        <string notr="true">color: rgb(238, 238, 236);
font: 75 14pt &quot;Height&quot;;
font-weight: bold;</string>
       </property>
       <property name="text">
        <string>Username:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" alignment="Qt::AlignHCenter">
      <widget class="QLineEdit" name="usernameLineEdit">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(238, 238, 236);</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="resultTableLabel">
    <property name="geometry">
     <rect>
      <x>7</x>
      <y>300</y>
      <width>160</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
font: 75 11pt &quot;DejaVu Sans&quot;;
font-weight: bold;
border-radius: 10px;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelTime">
    <property name="geometry">
     <rect>
      <x>385</x>
      <y>80</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(238, 238, 236);
color: rgb(239, 41, 41);
font: 75 11pt &quot;DejaVu Sans&quot;;
font-weight: bold;

border-radius: 10px;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>850</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
